### ISSUE TITLE
Move annotation properties into separate files/graphs

### ISSUE THREAD

**Issue by dr-shorthair** [qudt/qudt-public-repo#20]
Clean up the ontologies by

1. removing deprecated properties
2. moving annotation properties into separate files/graphs

(Copied from https://github.com/qudtdev/qudt-r2/issues/100)

**Comment by steveraysteveray** [comment]
I agree with your first point of getting rid of deprecated properties. But I'm wondering about the wisdom of your second suggestion. I say this because I recently finished consolidating all the separate graphs containing unit definitions. Different context, I realize, because here you are not talking about a duplicate definition. 
However, what problem are you trying to solve? Is it simply the size of the file? The units vocabulary file is still less than 1MB in size. Is it performance?
The downside of having separate graphs is more housekeeping, deciding whether a property is an annotation property or not, etc.
I'm willing to support your suggestion once I'm convinced - just not there yet.

**Comment by dr-shorthair** [comment]
Actually I was thinking more of the Ontology files. 
The core of the ontology is the classes and relationships and too many annotations can be a lot of clutter to fight through, particularly in the context of TopBraid UML-style diagrams.

**Comment by dr-shorthair** [comment]
I'm primarily looking for a better structure in the ontology files. 
Incidental properties can be housed in separate graphs, which then can be loaded only when needed.

**Comment by ralphtq** [comment]
Prefer not to have the schema split across schemas. SPARQL can do the filtering. In the Linked Data world there should also be a way to have filtering with content negotiation?
